cmd/query/app/server_test.go

```diff
@@ -664,7 +669,6 @@ func TestServerSinglePort(t *testing.T) {
 			}
 
 		}
-		wg.Done()
```

Comment:
> is it possible for the loop to exit without L665 condition being true?

I don't think so because all the servers all explicitly write `healthcheck.Unavailable` before the return.

Note I was copying the previous pattern - and this was the only case that had an explicit `wg.Done()` at the end.

I do think this is very ugly though so I'll have a go at simplifying this.